src/pages/bug-bounty/vulnerabilities-types/index.tsx

```diff
@@ -110,11 +110,7 @@ const VulnerabilitiesTypes = () => {
                                                         key={profit_table_idx}
                                                     >
                                                         <Header type="small" weight="normal">
-                                                            <Localize
-                                                                translate_text={
-                                                                    profit_table_item.title
-                                                                }
-                                                            />
+                                                            {profit_table_item.title}
```

Comment:
is this OK? :thinking: don't we need the translated value here ?

Reply:
yep, The value of title is `Localize` they passed localizes to localized :)))

Reply:
🧐